src/components/index.jsx

```diff
@@ -0,0 +1 @@
+export * from './Sum';
```

Comment:
Is it necessary?

Reply:
Wanted to try this export method so why not?

Reply:
okay)